dht.go

```diff
@@ -645,14 +650,17 @@ func (dht *IpfsDHT) peerFound(ctx context.Context, p peer.ID, queryPeer bool) {
 	} else if b {
 		select {
 		case dht.addPeerToRTChan <- addPeerRTReq{p, queryPeer}:
-		case <-dht.ctx.Done():
+		case <-ctx.Done():
```

Comment:
Was this change intentional? If so, we ought to change other places we use this embedded context (e.g,. `persistRTPeersInPeerStore`). Ideally we should try and remove the embedded context if possible.

Reply:
Removed, this was way too noisy.